src/test/test-flow.c

```diff
@@ -1151,12 +1152,14 @@ named_options_init_from_strv(void)
 {
     struct sol_flow_node_named_options named_opts;
     struct sol_flow_node_named_options_member *m;
+    struct sol_flow_node_type **node_type;
     int r;
 
+    node_type = sol_symbol_get("flow", "timer", SOL_FLOW_NODE_TYPE_TIMER);
     {
         const char *strv[] = { "interval=1000", NULL };
 
-        r = sol_flow_node_named_options_init_from_strv(&named_opts, SOL_FLOW_NODE_TYPE_TIMER, strv);
+        r = sol_flow_node_named_options_init_from_strv(&named_opts, *node_type, strv);
```

Comment:
well, you can have an intermediate variable, but anyway the macro for disabled modules would be:

```c
#define sol_symbol_get(..., sym, var) ({var = sym; 0})
```

that is, assign symbol to variable and return 0 (success).